source/operations/install-deploy-manage/expand-minio-deployment.rst

```diff
@@ -28,6 +28,24 @@ The procedure on this page expands an existing
 :ref:`distributed <deploy-minio-distributed>` MinIO deployment with an
 additional server pool. 
 
+Rebalancing Pools
```

Comment:
So this is just a thought - we're spending some cycles up front talking about rebalancing when our guidance is to not do this unless required.

We could instead move this to https://min.io/docs/minio/linux/operations/concepts.html under an H2 related to distributed MinIO, "How do I manage object distribution across a MinIO deployment?"

Then this is exceedingly useful, because we start with "You don't , we do", and follow up with "You can, but probably shouldn't."

Reply:
Actually, that's a really good idea. Moved to Concepts. Referencing that concepts section in the writing files section later on this page.